configuration/configuration-mainnet.yaml

```diff
@@ -1,5 +1,5 @@
 # global filter; messages must have at least this severity to pass:
-minSeverity: Notice
+minSeverity: Info
```

Comment:
Is it intentional to change this "main" configuration?

Reply:
the replay events are traced with severity "Info". wouldn't pass the filter.

Reply:
Sure. It's just that we're changing the main configuration. This is not the config used by QA and so? If it is, then we shouldn't change this on a whim.